orchestra/proc-macro/src/impl_channels_out.rs

```diff
@@ -81,14 +89,21 @@ pub(crate) fn impl_channels_out_struct(info: &OrchestraInfo) -> Result<proc_macr
 				&mut self,
 				signals_received: usize,
 				message: #message_wrapper,
+				priority: ChannelsOutPriority,
```

Comment:
Could we use `pub async fn send_and_log_error::<P: ChannelsOutPriority = Normal>(&mut self, signals_received: _, message: _) {..}`? I am not a fan of another runtime arg.

We don't need to introduce more priorities, but it'd allow us to extend in a non-breaking way - at least from a very high level point of view.

Reply:
Mainly API cosmetics that are difficult to undo once released.

Reply:
@drahnr is it what you mean?
https://github.com/paritytech/orchestra/pull/77/commits/51134cc377b91ca48f7aa5a88fdfd4e72a7d8062

I suppose that defaults in generic no more allowed in this way, so I didn't set defaults